libnavigation-base/src/main/java/com/mapbox/navigation/base/options/EHorizonOptions.kt

```diff
@@ -76,34 +76,60 @@ class EHorizonOptions private constructor(
      */
     class Builder {
 
-        private var length: Double = 500.0
-        private var expansion: Int = 0
-        private var branchLength: Double = 50.0
-        private var includeGeometries: Boolean = false
+        private var length: Double = DEFAULT_LENGTH
+        private var expansion: Int = DEFAULT_EXPANSION
+        private var branchLength: Double = DEFAULT_BRANCH_LENGTH
+        private var minTimeDeltaBetweenUpdates: Double? = DEFAULT_MIN_DELTA
 
         /**
-         * Override he minimum length of the EHorizon ahead of the current position.
+         * Override the minimum length of the EHorizon ahead of the current position.
+         * If negative default value will be used.
          */
         fun length(length: Double): Builder =
-            apply { this.length = length }
+            apply {
+                if (length < 0) throw IllegalArgumentException(
+                    "EHorizonOptions.length can't be negative."
+                )
+                this.length = length
+            }
 
         /**
          * Override the number of levels to expand.
+         * If negative default value will be used.
          */
         fun expansion(expansion: Int): Builder =
-            apply { this.expansion = expansion }
+            apply {
+                if (expansion < 0) throw IllegalArgumentException(
+                    "EHorizonOptions.expansion can't be negative."
+                )
+                this.expansion = expansion
+            }
 
         /**
          * Override the minimum length to expand branches.
+         * If negative default value will be used.
```

Comment:
not right comment here and in other builder's functions. Could you also mention about default value in each method

Reply:
ah, have forgotten to remove. will fix